src/cloud/aws/networking.md

```diff
@@ -9,7 +9,7 @@ The Technology Services Cloud Services team provides all AWS customers with a Vi
 
 The TAMU VPCs are available in the `us-east-1` and `us-east-2` regions with large public, private, and campus-accessible subnets in multiple availability zones.
 
-- **Public Subnets**: These subnets are used for resources that need to be accessible from the internet.
+- **Public Subnets**: These subnets are used for resources that need to be accessible from the internet. Don't get confused that the IP addresses encompassed are not themselves direcly publicly routable. It's about the route table and gateway setup.
```

Comment:
Let's link to the AWS document on subnet types instead: https://docs.aws.amazon.com/vpc/latest/userguide/configure-subnets.html#subnet-types

Reply:
I feel like linking to the doc is less obvious than what I wrote, but am willing to defer. I have updated to reflect what you asked for. But ask that you consider keeping what I wrote and adding the link for more details.